build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
     repositories {
-        jcenter()
+        mavenLocal()
         mavenCentral()
+        jcenter()
```

Comment:
According to Gradle documentation ([25.7. How dependency resolution works](https://docs.gradle.org/current/userguide/dependency_management.html#sec:dependency_resolution)): 
> Each repository is inspected in order [...] Modules from earlier repositories are preferred over modules in later repositories.

I added `mavenLocal()` as a first repository to search for required dependencies in your local Maven repository (if present) and if not found then proceed with the remote repositories to avoid unnecessary network traffic. This will speedup the build if you have local Maven repository on your machine already.

Then I moved `mavenCentral()` between `mavenLocal()` and `jcenter()` to make it look nicer (Maven repos grouped together).